benchmarking/benchmarks/dataset.py

```diff
@@ -36,9 +37,15 @@ class Adding5Params:
     # latter case asv will run the benchmark for all the combinations of the
     # values
     params = [
-        {'n_values': 10000, 'n_times': 2},
-        {'n_values': 100, 'n_times': 200}
+        {'n_values': 10000, 'n_times': 2, 'paramtype': 'array'},
+        {'n_values': 100, 'n_times': 200, 'paramtype': 'array'},
+        {'n_values': 10000, 'n_times': 2, 'paramtype': 'numeric'},
+        {'n_values': 100, 'n_times': 200, 'paramtype': 'numeric'},
     ]
+    # we are less interested in the cpu time used and more interested in
+    # the wall clock time used to insert the data so use a timer that measures
+    # wallclock time
```

Comment:
wallclock time of _this process_, you mean, right?

Reply:
No Wallclock time of a process is IMHO a undefined.There is only one wallclock time